CloudFormation/vpc/freetier/ec2.cf

```diff
@@ -16,6 +16,15 @@ Parameters:
     Description: The EC2 Key Pair to allow SSH access to the instance
     Type: 'AWS::EC2::KeyPair::KeyName'
 
+  DeployNameParameter:
+    Description: Choose a deploy to run on your host.
+    Type: String
+    Default: baseline/deploy.sh
+    AllowedValues:
```

Comment:
I should note the only one that works is "openarena/deploy.sh" as the other two do not exist, but as you can see it will be trivial to add as many as you like... just put them in a subfolder in the s3 bucket and list them here...